web/pandas/about/team.md

```diff
@@ -52,7 +52,15 @@ The project governance is available in the [project governance page]({{ base_url
 ### {{ workgroup.name }}
 
 <ul>
-    <li><b>Contact:</b> <a href="mailto:{{ workgroup.contact }}">{{ workgroup.contact }}</a></li>
+    <li><b>Contact:</b>
+        <a id="{{ ''.join( workgroup.name.split(' ') ) }}" href="mailto:{{ 'asp.' + workgroup.contact }}">{{ 'asp.' + workgroup.contact }}</a>
```

Comment:
Maybe I'm missing something, but splitting by space and then joining should be the same as replacing, and jinja has a replace filter we can use.

Unrelated, the asp can leave directly in the html, a bit simpler than concatenating it to the variable first, no?

```suggestion
        <a id="{{ workgroup.name|replace(' ', '-') }}" href="mailto:asp.{{ + workgroup.contact }}">asp.{{ workgroup.contact }}</a>
```

Reply:
Hi , I have tested this and it works , 

changes suggested seem fine, but we would need to do the same change in script tag as well. I'll suggest that change in a few minutes.

`